templates/pulpcore-api.service.erb

```diff
@@ -11,7 +11,7 @@ User=<%= scope['pulpcore::user'] %>
 Group=<%= scope['pulpcore::group'] %>
 WorkingDirectory=<%= scope['pulpcore::user_home'] %>
 RuntimeDirectory=pulpcore-api
-ExecStart=/usr/libexec/pulpcore/gunicorn pulpcore.app.wsgi:application \
+ExecStart=/usr/bin/pulpcore-api \
```

Comment:
This will require adjustments in https://github.com/theforeman/foreman/blob/1eb33c7eb13e6f547f8bcb21ccf5969e8b876e98/extras/pcp/foreman-hotproc.conf#L9-L10 to make PCP pick up the new processes.

(Just add the new names to the list, so the old ones keep working, I guess)

Reply:
https://github.com/theforeman/foreman/pull/9898